ha/core/controllers/pcs/pcs_controller.py

```diff
@@ -72,6 +71,27 @@ def clean_failure_count(self, node_id):
         _output, _err, _rc = self._execute.run_cmd(const.PCS_NODE_CLEANUP.replace("<node>", node_id),
                                                    check_error=False)
 
+    def auth_node(self, node_id, cluster_user, cluster_password):
+        """
+        Auth node to add
+        """
+        try:
+            _output, _err, _rc = self._execute.run_cmd(const.PCS_CLUSTER_NODE_AUTH.replace("<node>", node_id)
+                                                       .replace("<username>", cluster_user)
+                                                       .replace("<password>", cluster_password))
+        except Exception as e:
+            raise ClusterManagerError(f"Failed to authenticate node : {node_id} with reason : {e}")
+
+    def get_cluster_size(self):
```

Comment:
convert it to protected function

from `get_cluster_size` to `_get_cluster_size`

Reply:
Done